src/patient-registration/patient-registration-context.ts

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+type PatientRegistrationContextProps = {
+  identifierTypes: Array<any>;
+  values: any;
+  validationSchema: any;
+  setValidationSchema: (value: any) => void;
+  inEditMode: boolean;
+  fieldConfigs: any;
```

Comment:
Is this any? Or maybe some `Record<U, T>`?

Reply:
Actually the structure of this object cannot be known before hand as field configs are not uniform amongst all fields. For example the name field has a config to turn of middle name and it looks like this

```javascript
fieldConfigs: {
    name: {
        displayMiddleName: true,
     }
}
```
And other fields might have configurations that have different shapes too. Thats why I used `any`.

Reply:
Well, but from your example it seems that this is at least `Record<string, any>` if not even `Record<string, Record<string, boolean>>` or so. There is some structure here at work and I'd like to know it.

If it would be indeed `any` as you suggest it can be `string`, `boolean`, `number`, ... -> but I doubt that.